recipes/xtr/all/conanfile.py

```diff
@@ -39,6 +58,10 @@ def validate(self):
             raise ConanInvalidConfiguration(f"Unsupported compiler={self.settings.compiler}")
         if self.settings.arch not in ("x86_64", ):
             raise ConanInvalidConfiguration(f"Unsupported arch={self.settings.arch}")
+        if tools.Version(self.version) < "2.0.0" and self.settings.compiler == "clang" and self.settings.compiler.libcxx == "libc++":
+            raise ConanInvalidConfiguration(f"Use at least version 2.0.0 for libc++ compatibility")
+        if self.get_option("enable_io_uring_sqpoll") and not self.get_option("enable_io_uring"):
+            raise ConanInvalidConfiguration(f"io_uring must be enabled if io_uring_sqpoll is enabled")
```

Comment:
```suggestion
            raise ConanInvalidConfiguration(f"io_uring must be enabled if io_uring_sqpoll is enabled")
        if self.options.get_safe("sink_capacity_kb") and not str(self.options.get_safe("sink_capacity_kb")).is_digit():
            raise ConanInvalidConfiguration(f"The option 'sink_capacity_kb' must be an integer.")
```
if sink_capacity_kb is defined and is not an digit, the user is wrong

Reply:
I've added this via "commit suggestion" but GitHub doesn't remove it from the pull